tests/unit/include/bout/test_region.cxx

```diff
@@ -242,7 +242,7 @@ TEST_F(RegionTest, regionLoopAllSerial) {
   auto region = mesh->getRegion3D("RGN_ALL");
 
   Field3D a = 0.0;
-  BLOCK_REGION_LOOP_SERIAL(region, i, a[i] = 1.0;);
+  BLOCK_REGION_LOOP_SERIAL(i, region) { a[i] = 1.0; }
```

Comment:
Is there any benefit to also having a `BLOCK_REGION_LOOP_SECTION` version of these?

Reply:
Yes, possibly -- `BLOCK_REGION_LOOP` is now implemented in terms of `BLOCK_REGION_LOOP_SECTION`, so in some respect it is already tested. But it wouldn't hurt to have an explicit test